01 - Community/People/trashhalo.md

```diff
@@ -21,7 +21,6 @@ publish: true
 %% Begin Hub: Released contributions %%
 ### Plugins
 - [[extract-url|Extract url content]]
-- [[todo-txt|Todo.txt support]]
```

Comment:
I wouldn't remove this. This should go to inactive/unmaintained.

Reply:
Done by reverting that commit.